corgi/tasks/yum.py

```diff
@@ -19,9 +25,15 @@
     retry_kwargs=RETRY_KWARGS,
     soft_time_limit=settings.CELERY_LONGEST_SOFT_TIME_LIMIT,
 )
-def fetch_unprocessed_yum_relations(force_process: bool = False) -> int:
+def fetch_unprocessed_yum_relations(force_process: bool = False, created_since: int = 0) -> int:
```

Comment:
Perhaps call it `days_created_since` so it's immediately clear that it expects an int value for number of days? Or change it to expect the timedelta value as you did in `fetch_unprocessed_relations()`.

Reply:
I renamed it to "days_created_since" so we can easily set it to 0 and get the "look up last success" behavior, else explicitly force processing a certain number of days.

But I think when I force-pushed it cleared your review. Or maybe you didn't approve yet? I'll let this sit a bit while doing some paperwork - will merge tomorrow if no further feedback from you / Jim I guess. Thanks for reviewing!